from app.api.v1.features.exceptions import GiftAppError


class PastEventError(GiftAppError):
    pass